netlify.toml

```diff
@@ -0,0 +1,9 @@
+[build]
+  publish = "public"
+  command = "gatsby build"
+
+[[redirects]]
+  from = "https://blog.cilium.io/*"
+  to = "https://cilium.io/blog/:splat"
+  status = 200
```

Comment:
@brb do you want to serve the blog from https://blog.cilium.io/ without changing URL in the status bar, or do you want https://blog.cilium.io/ to redirect to https://cilium.io/blog/ ?

Reply:
Either works :pray: